CMakeLists.txt

```diff
@@ -150,7 +150,14 @@ add_subdirectory(lib)
 add_subdirectory(exe)
 
 # Add test executables for unit testing
-add_test(NAME "library\\ tests" COMMAND libfacter_test)
+add_test(NAME "libfacter\\ tests" COMMAND libfacter_test)
+if (RUBY_FOUND)
+    if (WIN32)
+        add_test(NAME "libfacter\\ specs" COMMAND bundle.bat exec rspec WORKING_DIRECTORY "${PROJECT_SOURCE_DIR}/lib")
```

Comment:
It is a CTest issue.  It finds the .bat file but can't execute it without extension.